tests/unit/s2n_tls13_handshake_state_machine_test.c

```diff
@@ -516,24 +624,39 @@ int main(int argc, char **argv)
         /* Ensure OCSP_STATUS is set by setting the connection status_type */
         conn->status_type = S2N_STATUS_REQUEST_OCSP;
 
-        /* Verify setup: tls1.2 DOES set the flags */
+        /* Verify that tls1.2 DOES set the flags allowed by tls1.2 */
```

Comment:
For future readability: could we update this comment to explicitly call out the flags allowed by TLS1.2 here?

Reply:
See https://github.com/awslabs/s2n/pull/2296#discussion_r492424779. I think the list of EXPECT_TRUEs lists the flags allowed by TLS1.2 pretty clearly and I prefer a single source of truth.